README.md

```diff
@@ -2,3 +2,5 @@ git-hw-to-fork
 ==============
 
 The repository for students to fork (and submt a pull request) in the first git HW
+
+![community party hard](http://i.giphy.com/zPVRKhPsUP5lK.gif)
```

Comment:
"That's why I was willing to change for you guys. Because when you really know who you are and what you like about yourself, changing for other people isn't such a big deal."